src/ImageSharp/Processing/Processors/Transforms/Resize/ResizeProcessor{TPixel}.cs

```diff
@@ -168,10 +172,12 @@ public void ApplyTransform<TResampler>(in TResampler sampler)
             Rectangle sourceRectangle,
             Rectangle destinationRectangle,
             Rectangle interest,
-            bool compand)
+            bool compand,
+            bool premultiplyAlpha)
         {
-            PixelConversionModifiers conversionModifiers =
-                PixelConversionModifiers.Premultiply.ApplyCompanding(compand);
+            PixelConversionModifiers conversionModifiers = premultiplyAlpha ?
+                PixelConversionModifiers.Premultiply.ApplyCompanding(compand) :
+                PixelConversionModifiers.None.ApplyCompanding(compand);
```

Comment:
Applying compand to the None modifier. Not sure if correct.

Reply:
It's OK, although not super happy about how it looks now. Would replace the `ApplyCompanding` helper with something like:

```C#
public static PixelConversionModifiers GetModifiers(bool compand, bool premultiplyAlpha = true);
```

Reply:
I split it out into a separate private function. I hope that's what you meant.

Reply:
I'm ok with the private function for now. This is the only place that uses it.